packages/bruno-app/src/components/Sidebar/Collections/Collection/CollectionItem/index.js

```diff
@@ -189,16 +189,27 @@ const CollectionItem = ({ item, collection, searchText }) => {
       toast.error('URL is required');
     }
   };
+  
   const viewFolderSettings = () => {
-    dispatch(
-      addTab({
-        uid: uuid(),
-        collectionUid: collection.uid,
-        folderUid: item.uid,
-        type: 'folder-settings'
-      })
-    );
+    const newTabUid = uuid();
```

Comment:
Instead of generating a new UUID for the tab, could the folderUid as the TabUid? I think this is also done for the Request tabs.

Reply:
Yes, we can do that too. Thanks, @Its-treason for reviewing.

Reply:
Thanks @Its-treason , We have made this change.